tulibraries/tul_cob
# Fix shib logout not redirecting properly.

REF BL-224

Redirects to shib url on logout.

QA Steps
===
- Log in as a user (you'll need to temporarily enable forms.)
- Click Log Out button
- [ ] Verify that you get a redirect to a shib URL.  Note you will get error on local because your local does not support shib.
